Song_sort_Genre.css

```diff
@@ -1,64 +1,38 @@
+/* Import  font from google fonts*/
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap');
+
 body{
 	margin:0;
 	padding:0;
 	background:white;
-	overflow: scroll;
-	overflow-x: hidden;	
+	
 }
 
 .grid-container{
 	display: grid;
-	grid-template-columns: 15% 10% 75%;
-	grid-template-rows: auto auto auto auto auto;
+	grid-template-columns: 15% 35% 50%;
+	grid-template-rows: 58px auto auto auto auto;
 	height: 100vh;
 }
 
-.img_1{
-	grid-column-start: 1;
-	grid-column-end: span 4;
-	height:300px;
-	background:red;
-}
-
-.img_1 img{
-	display:block;
-	margin-left:auto;
-	margin-right:auto;
-	height:300px;
-	width:100%;
-}
-
-.footer{
-	background-color: white;
-	grid-column-end: span 4;
-	padding-top:25px;
-	text-align: center;
-	color:black;
-	height:50px;
-}
-
 .navigation{
 	box-sizing: border-box;
 	grid-column-start: 1;
 	grid-column-end: span 4;
-	margin-top:-10px;
-	height:60px;
-	background-color:red;
 }
 
 
 nav{
 	width:100%;
-	background:white;
+	background-color: #0C59CF;
 	overflow:auto;
-	position: static;
 	
 }
 
-
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
```

Comment:
21/9: I change the margin for the link in the nav bar so the position in right for my link because the old margin was wrong because when I added the drop menu function I drop down menu column was shift and doesn't match. So I need to reposition of those column, but it only work if the type of screen was in a specific size which the the screen size of my laptop since I make it to be in with my screen size. So I decided to set a difference margin so I don't need to to reposition of the drop down menu column and it will be consistent for all screen.